yt_dlp/extractor/newgrounds.py

```diff
@@ -128,20 +132,26 @@ def _real_extract(self, url):
             (r'<dt>\s*Uploaded\s*</dt>\s*<dd>([^<]+</dd>\s*<dd>[^<]+)',
              r'<dt>\s*Uploaded\s*</dt>\s*<dd>([^<]+)'), webpage, 'timestamp',
             default=None))
-        duration = parse_duration(self._search_regex(
-            r'(?s)<dd>\s*Song\s*</dd>\s*<dd>.+?</dd>\s*<dd>([^<]+)', webpage,
+        duration = parse_duration(self._html_search_regex(
+            r'"duration"\s*:\s*["\']?([\d]+)["\']?,', webpage,
             'duration', default=None))
 
-        view_count = parse_count(self._html_search_regex(r'(?s)<dt>\s*Views\s*</dt>\s*<dd>([\d\.,]+)</dd>', webpage,
-                                                         'view_count', fatal=False, default=None))
+        view_count = parse_count(self._html_search_regex(
+            r'(?s)<dt>\s*Views\s*</dt>\s*<dd>([\d\.,]+)</dd>', webpage,
+            'view count', default=None))
 
-        filesize_approx = parse_filesize(self._html_search_regex(
-            r'(?s)<dd>\s*Song\s*</dd>\s*<dd>(.+?)</dd>', webpage, 'filesize',
+        filesize = parse_filesize(self._html_search_regex(
+            r'"filesize"\s*:\s*["\']?([\d]+)["\']?,', webpage, 'filesize',
             default=None))
+
+        video_type_description = self._html_search_regex(
+            r'"description"\s*:\s*["\']?([^"\']+)["\']?,', webpage, 'filesize',
+            default=None)
+
         if len(formats) == 1:
-            formats[0]['filesize_approx'] = filesize_approx
+            formats[0]['filesize'] = filesize
```

Comment:
are you sure?

Reply:
Yes, here are files:
```
$ du -b ~/Downloads/*.mp3
4557743	/home/pogaca/Downloads/B7 - BusMode - 549479.mp3
2192718	/home/pogaca/Downloads/Zombie Wind - 500392.mp3
```
File sizes detected by yt-dlp:
![file1](https://user-images.githubusercontent.com/54671367/132954763-4d5574a2-d0b0-466d-8337-ee0b3fbe7d76.png)
![file2](https://user-images.githubusercontent.com/54671367/132954764-88adfd0a-7c27-485b-896e-898689e21d2a.png)